.github/CONTRIBUTING.md

```diff
@@ -142,7 +149,10 @@ builder failures, you may be experiencing a [test suite
 issue](https://github.com/zfsonlinux/zfs/issues?q=is%3Aissue+is%3Aopen+label%3A%22Test+Suite%22).
 There are also various [buildbot options](https://github.com/zfsonlinux/zfs/wiki/Buildbot-Options)
 to control how changes are tested.
-* All proposed changes must be approved by a ZFS on Linux organization member.
+* If your pull request has not yet passed the test suite,
+please think about submitting it as a draft PR instead.
```

Comment:
Better!  What do you think of this alternate wording which instead explains why you may want to open the PR as a draft (and links to the documentation).

* Consider opening a [draft PR]( https://github.blog/2019-02-14-introducing-draft-pull-requests/) for proposed changes which you would like to discuss or require additional testing.  Once the PR is in good shape you can remove the draft status which allows it to be merged.

Reply:
@behlendorf An improvement ofcourse...
Albeit a bit... longer and thus harder to read, but an inprovement none-the-less :)